app/src/main/java/org/hwyl/sexytopo/control/activity/SexyTopoActivity.java

```diff
@@ -62,11 +62,12 @@
 public abstract class SexyTopoActivity extends AppCompatActivity {
 
 
-    protected static SurveyManager dataManager;
```

Comment:
Android studio claimed this was a memory leak

Reply:
Ah yep, it does whinge about that. It's because it's holding a reference to a `Context`. I don't think it's an issue - the only context we're setting is an `ApplicationContext`, which is (apparently) fine.

Looks like not a problem making this non-static though.